### DISCUSSION TITLE
Using a RF filter

### DISCUSSION THREAD

**Discussion by Gazman429** [jvde-github/AIS-catcher#319] [General]
Thought I'd share one of my experiences.
My AIS station normally has had reasonable coverage until a couple of months ago when I was doing routine checks on my radio tower which saw it having to be halved in height from 20 meters to 10 meters due to corrosion issues on the top half.
I had reduced coverage as a result, but it also high lighted an issue that really hasn't bothered me to much and ignored until now.
Good old RF interference and intermodulation, I live in a RF hot spot area, 3 high power VHF pager sites, 1 VHF trunking radio site, 2 VHF marine repeaters, 1 amateur VHF repeater high power and a number of amateur home stations and now with an addition of a new VHF telemetry repeater nearby it all started having an impact on receiving AIS signals and RTL dongles have a very broad front end making them quite prone to intermod and adjacent channel interference.
Being involved in the radio communications industry for 12 years I acquired some retired but fully functional equipment here and there, some of them being RF cavity filters.
I tuned and put one to use it made a big difference, I'm not in a high vessel traffic number area.
Over a four day period, I ran a side by side test.
Two RTL dongles same type, Antenna - one VHF collinear 3dBd gain, 30 meters of LDF 450 coax cable, 1 RFI receive multicoupler set at 1.5db gain on each output port, 1 windows computer, RTL dongles both had the same settings - gain 38.6 - bandwidth (-a 192k) ,  -m 4, (RF cavity filter loss 0.6db 15 year old unit), day 1 both no RF filter and both had very similar results, day 2 - 1 with RF filter and one without, the receiver with filter had an increase of vessel count and decode, day 3 swap RF filter to the other dongle same results in reverse, day 4 let it run a full 24 hours.
Following screen shots, left is with no RF filter and right is with RF filter, I don't have a 162mHz saw filter but if I can find a good source I would run a 3 way side by side test comparison.
![2024-10-16 08 05 03](https://github.com/user-attachments/assets/12486cec-f64e-4033-9035-4daae1078dd1)
![2024-10-16 08 02 46](https://github.com/user-attachments/assets/46d7f537-a09b-41d9-9eac-11ca6e0bd766)
![2024-10-16 08 04 39](https://github.com/user-attachments/assets/15391292-f58c-44ec-900c-3d0c327e237d)

**Comment by jvde-github** [comment]
Nice, thanks for sharing. I live close to a radio tower as well and for me the GPIO AIS filter and ShipXplorer AIS dongle made a big difference (also running with -a 192K somewhat). I also felt the Airspy HF+ with sampling at 192K was a bit more robust in this set up. In another location with a clean signal the RTL-SDR V3 works best.

But don't have a radio background so have not looked at other options. Thanks for sharing, good read.

**Reply by Gazman429** [reply]
With AIS after doing a little back ground research and finding the max RF power output of the AIS transmitters and 4 classes of max RF power and averaging common transmission line losses more so on smaller vessels and licencing checks it's quite a challenge pulling weak signals from all the noise, for those not in the know, Max transmit power - Class A (SOTDMA) 15 watts but found many on the market with 12 watt transmitters - Class B+"SO" (SOTDMA) AIS transmitters 5 watts - Class B"CS" (CSTDMA) AIS transmitters 2 watts (but found nearly all in this class only output 1.9 watts) and fixed base stations SOTDMA 12 watts, non of this also takes into account transmission line losses, poor vessel RF maintenance etc which can result in only 1 watt of power leaving the antenna on a Class B"CS" AIS transmitter, limiting interference and a bit of experimenting with various settings to get the best results is achievable.
  In my time working in the RF industry and a little side line work after I also spent around half of that doing real world RF coverage area test land and water from 1 watt to 100,000 watt transmitters, and for 4 years spent 220 days each year many long hours driving and just under a million kilometres travelled getting both wide area and local area coverage, but it was a very good experience to involved in and has lead to all these computer based RF coverage systems to be what they are today from real world data and what trees and buildings around me affect how well I receive radio signals at my location.
